source/agora/test/Base.d

```diff
@@ -1210,14 +1210,20 @@ public class TestAPIManager
         throw new Exception(format!"TX (%s) was not externalized in %d blocks!"(tx_hash, n_blocks));
     }
 
-    /// Ensure that a transaction is accepted by a node (put in the tx pool)
-    public void ensureTxInPool (Idxs) (in Transaction tx, Idxs indices)
+    /// Post a transaction to clients and ensure it is accepted by them (put in the tx pool)
+    public void postAndEnsureTxInPool (Idxs) (in Transaction tx)
+    {
+        this.postAndEnsureTxInPool(tx, iota(this.clients.length);
+    }
+
+    /// Ditto
+    public void postAndEnsureTxInPool (Idxs) (in Transaction tx, Idxs indices)
```

Comment:
How about reordering the parameters like other functions that have the `indices` as the first parameter.

Reply:
Yes good idea.

Reply:
I did this in the last commit so that the changes are easier to follow.